src/client/src/apps/MainRoute/widgets/blotter/components/blotterUtils.ts

```diff
@@ -44,8 +44,10 @@ const getStatusIndicatorClass = (trade: Trade) => {
 export const DEFAULT_COLUMN_DEFINITION: ColDef = {
   menuTabs: ['filterMenuTab'],
   suppressSizeToFit: true,
-  suppressFilter: false,
+  filter: false,
```

Comment:
this looks like the opposite of what it was before, is that intentional?

Also if you have this here, do you need the repeated `filter: false`s below

Reply:
You're right, I don't need `filter: false` on `Trade Date` && `Value Date` columns